### PR TITLE
Condenses model to the left and other changes

### LINKED ISSUE #770: Sangat Sync: Condense the modal
Remove the video button (we will add to help section) and also the big icon in the modal.  Move the pairing code to the left (slightly larger and bold text) and keep the instructions text to the right of that.

The Stop Session and Present button should be under the code.

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Screenshot 2019-10-02 at 7 32 36 PM
Screenshot 2019-10-02 at 7 41 51 PM

Fixes #770  #771 #773 #769 #774 #772

### CODE REVIEW

File: www/js/toolbar.js [hunk 1]
```diff
@@ -49,9 +49,11 @@ const remoteSyncInit = async () => {
     code = await tryConnection();
     if (code) {
       document.querySelector('.sync-code-num').innerText = code;
+      document.querySelector('#tool-sync-button').setAttribute('title', code);
     }
   } else {
     document.querySelector('.sync-code-num').innerText = '...';
+    document.querySelector('#tool-sync-button').setAttribute('title', '...');
```
Comment (tsingh777) [inline www/js/toolbar.js]: i'd leave it blank

File: www/js/toolbar.js [hunk 2]
```diff
@@ -78,36 +80,39 @@ const switchFactory = (id, label, inputId, clickEvent, defaultValue = true) =>
...
   ]);
 
+const syncToggle = async (forceConnect = false) => {
+  if (isConntected && !forceConnect) {
+    isConntected = false;
+    onEnd(code);
+    code = '...';
+    global.controller.sendText('');
+    document.querySelector('.sync-code-num').innerText = '...';
+    document.querySelector('#tool-sync-button').setAttribute('title', '...');
```
Comment (tsingh777) [inline www/js/toolbar.js]: leave title blank

File: www/js/toolbar.js [hunk 3]
```diff
@@ -78,36 +80,39 @@ const switchFactory = (id, label, inputId, clickEvent, defaultValue = true) =>
...
-  ]),
-  h('div.right-sync-content', [
-    h('div.sync-code-label', 'Your unique sync code is'),
-    h('div.sync-code-num', code),
     h(
       'div.sync-code-desc',
       'Enter this code on sttm.co/sync to follow along SikhiToTheMax on any web browser (including mobile). Press the "Present" button to display the pairing code for the sangat.',
     ),
+  ]),
+  h('div.right-sync-content', [
```
Comment (tsingh777) [inline www/js/toolbar.js]: can we make it so the popout closes clicking outside of the content? having the go the close button is far mouse travel.

Comment (maneetpaul) [reply www/js/toolbar.js]: +1

**Pull request merged**